altair/examples/stripplot.py

```diff
@@ -0,0 +1,39 @@
+"""
+Stripplot
+-----------------
+This example shows how to make a Stripplot.
+"""
+# category: scatter plots
+import altair as alt
+from vega_datasets import data
+
+source = data.movies()
+
+stripplot = (
+    alt.Chart(source, width=40)
+    .mark_circle(size=8)
+    .encode(
+        alt.X(
+            shorthand="jitter:Q",
```

Comment:
Please pass the shorthand as a positional arg rather than by keyword.

Reply:
fixed